source/Patches/Roles/Role.cs

```diff
@@ -547,6 +586,20 @@ private static void UpdateMeeting(MeetingHud __instance)
                     }
                     else
                     {
+                        //Remove red color for impostors in meeting if anonymous impostor
+                        var playerChar = PlayerControl.AllPlayerControls.ToArray()
+                        .FirstOrDefault(x => x.PlayerId == player.TargetPlayerId);
```

Comment:
Pretty sure this already exists? Utils.GetPlayerById or something? Wouldn't be surprised if the PlayerControl object is attached to the player state somehow too.

Reply:
no, i don't know why this comment was here

Reply:
This is referring to the code to get a player from a player id. This is the kind of thing we should write once and re-use everywhere.

Reply:
if you say so, you are the expert for that